src/resources/reactions.js

```diff
@@ -0,0 +1,12 @@
+const api = require('../api');
+
+function reactions(config) {
+  const url = `${config.apiURL}/reactions`;
+  const call = (method, params) => api(url, config, method, params);
+  return {
+    add: params => call('POST', params),
```

Comment:
We should also have a method for GETting the existing reactions on a message, perhaps.